react-payments/src/component/SavedCardList/SavedCardList.component.tsx

```diff
@@ -4,24 +4,22 @@ import CardPreview from "component/CardPreview/CardPreview.component";
 import SkeletonCardBox from "component/common/SkeletonCard/SkeletonCard.component";
 
 import useFetch from "hooks/useFetch";
-import { API_URL, REDUCER_TYPE } from "constants";
-import { CardDataContext } from "provider/CardDataProvider";
+import { API_URL } from "constants/index";
+import { CardDataContext, initCardDataAction } from "provider/CardDataProvider";
 
 const SavedCardList = () => {
   const { data, loading } = useFetch(`${API_URL}/api/cards`);
-  const { cardData, dispatch } = useContext(CardDataContext);
-
+  const cardDataContext = useContext(CardDataContext);
+  if (!cardDataContext) {
+    throw new Error("Cannot find CardDataContext");
+  }
+  const { cardData, dispatch } = cardDataContext;
```

Comment:
```suggestion
  const { cardData, dispatch } = useCardDataContext;
```
context 호출하는 함수에서 context가 없으면 에러를 던지는 구문이 가독성을 해친다는 느낌을 받았는데 별도로 커스텀 훅을 만들어서 내부에서 에러를 던지는게 어떨까요?